CIP-????/README.md

```diff
@@ -0,0 +1,172 @@
+
+## Goals
+
+Governance on Cardano in this CIP is defined as on-chain decision making that is automatically executed when some threshold of approval by human actors is met. Each threat or vulnerability will be described as a "threat", "description", and the suggested mitigiation technique will be described as a "recommendation" for each type of threat identified in this document. All known threats and mitigations will be made available to public as broadly as possible. The more voters, developers, and governance actors are aware of threats the more likely they are to be identified and mitigated before damage occurs.
+
+
+On rare occasion a threat may be identified that requires responsible reporting procedures in order to prevent exposing an active vulnerability to bad actors who may aim to exploit the vulnerability. In the event responsible reporting is required, notify the appropriate group or entity with the responsibility and resources to mitigate such threat in a confidential manner.
+
+Assumption: There are no known software back doors. Unknown software vulnerabilities may be discovered.
+
+### #1: Unknown software vulnerability affecting governance is discovered.
+
+Recommendation: The person discovering the vulnerability should document the circumstances and details in which the vulnerability is present and how to reproduce the vulnerability. Then report the vulnerability in a confidential manner using a secure method to the developer team with the resources to correct the vulnerability.
+
+### #2: Emergency group to support the disaster recovery plan.
+  -attacks on persons if known
+  -we don’t want attackers know who that group is?
+
+Recommendation:
+
+### #3: Emergency change that needs to be done quickly.
+  -a nuclear option
+  -change requires “just do it” 
+  -a security that fix must be implemented but CANNOT be communicated publicly
+
+Recommendation:
+
+### #4: Operational Security (OPSEC) of the committee.
+  -physical security
+  -digital security
+
+Recommendation:
+
+### #5: Committee keys lost or stolen or committee keys sold to a bad actor.
+
+Recommendation:
+
+### #6: DRep keys lost or stolen or DRep keys sold to a bad actor.
+
+Recommendation:
+
+### #7: Lack of governance for an extended period of time.
+  -Uncontrolled hard forks by SPOs
+  -Risk of no confidence
+
+Recommendation:
+
+### #8: Lack of trust in voter tooling.
+
+Recommendation:
+
+### #9: Trust in the wallets.
+  -wallet voting attack vector
+  -hardware wallet support
+
+Recommendation:
+
+
+### #10: Corruption - in any form, any number of actors.
+  -Code of Conduct mitigation
+  -on-chain mitigation
+
+
+Recommendation:
+
+### #11: Collusion - the bad version of collaboration to do something illegal or undesirable.
+
+Recommendation:
+
```

Comment:
If this is collusion on the part of any committees members or DReps, it should result in loss of collateral by committee vote. Evidence needs to be presented in a reasonable and articulable way to a body (i.e. Oversight & Accountability Committee), with specific constitutional citations that explains the behavior and/or actions. Voting on such an action must be constitutionally based, not emotionally based.